renderer/components/ui/AutocompleteSelector.tsx

```diff
@@ -171,12 +184,12 @@ export default function AutocompleteSelector({
           />
         </Combobox.Button>
 
-        {filteredList?.length > 0 ? (
+        {fullSuggestionsList?.length > 0 && (
           <Combobox.Options className="absolute z-10 w-full py-1 mt-1 overflow-auto text-base bg-white rounded-md shadow-lg max-h-40 ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm dark:bg-dark-container dark:shadow-lg dark:shadow-slate-900">
             <div className="block text-xs text-gray-500 text-center">
               tab to choose
             </div>
-            {filteredList?.map((item, i) => (
+            {fullSuggestionsList.map((item, i) => (
```

Comment:
It'd be nice to move this map logic outside of the markup. it's nice to have a loop logic outside of a markup because a code becomes clear due to such actions.

Reply:
You can use a function for this case.